.travis.yml

```diff
@@ -0,0 +1,201 @@
+---
+sudo: required
+dist: trusty
+
+services:
+  - docker
+
+script:
+  # Build docker image
+  - cd 2.6/
```

Comment:
I'm still unsure about this - the benefit of having commands inside the travis.yml is to see the return codes step by step. We could maybe achieve this with a "set -e". I may keep it like that for the moment and refactor later